src/uu/uuidgen/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "uu_uuidgen"
+version = "0.0.1"
+edition = "2021"
+
+[lib]
+path = "src/uuidgen.rs"
+
+[[bin]]
+name = "uuidgen"
+path = "src/main.rs"
+
+[dependencies]
+clap = { workspace = true }
+rand = { workspace = true }
+thiserror = { workspace = true }
+uucore = { workspace = true }
+uuid = { workspace = true, features = ["v1", "v3", "v4", "v5"] }
+
+[target.'cfg(target_os = "windows")'.dependencies]
+windows = { workspace = true, features = ["Win32_NetworkManagement_IpHelper", "Win32_NetworkManagement_Ndis", "Win32_Networking_WinSock"] }
+
+[target.'cfg(any(target_os = "android", target_os = "linux", target_os = "freebsd", target_os = "dragonfly", target_os = "netbsd", target_os = "openbsd", target_os = "ios", target_os = "macos", target_os = "watchos", target_os = "tvos", target_os = "visionos", target_os = "illumos", target_os = "solaris"))'.dependencies]
```

Comment:
I think you could use something like:

[target.'cfg(not(target_os = "windows"))'.dependencies]